Finmer.Game/Views/CharCreateBasic.xaml.cs

```diff
@@ -35,15 +35,30 @@ private void CharCreateViewBase_Loaded(object sender, RoutedEventArgs e)
         {
             txtName.Text = InitialSaveData.GetString("name");
             cmbSpecies.Text = InitialSaveData.GetString("species");
-            optGenderMale.IsChecked = InitialSaveData.GetString("gender").Equals("Male");
-            optGenderFemale.IsChecked = !optGenderMale.IsChecked;
+            if (!String.IsNullOrWhiteSpace(InitialSaveData.GetString("gender")))
+            {
+                optGenderMale.IsChecked = InitialSaveData.GetString("gender").Equals("Male");
+                optGenderFemale.IsChecked = !optGenderMale.IsChecked;
+            }
             m_Setup = false;
 
             // Apply some default settings to enable quickly clicking through to the game
             //if (GameController.DebugMode)
             {
                 txtName.Text = "Snack";
                 cmbSpecies.SelectedIndex = CoreUtility.Rng.Next(cmbSpecies.Items.Count);
+                //make a coin flip to see if it's male or female randomly selected, then set the gender accordingly.
```

Comment:
Some style comments:
- please start comments with a space followed by a capital, so `// Make ...`.
- local variables should use lower_snake_case, so `gender_flip` instead of `genderFlip`. (Check [contributing.md](https://github.com/pileofwolves/finmer/blob/master/.github/CONTRIBUTING.md) for naming guidelines.)